CMakeLists.txt

```diff
@@ -92,7 +92,10 @@ if(XLINK_ENABLE_LIBUSB)
         if(WIN32 AND NOT MINGW)
```

Comment:
```suggestion
        if(WIN32 AND CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
```

Reply:
Probably above is the way to go then, as it seems that DELAYLOAD is MSVC feature (and not "everyone's but Clang's feature)
Also, the delayimp.lib then isn't required.